File: app/src/main/java/ca/etsmtl/etsmobile3/data/model/Etudiant.java
package ca.etsmtl.etsmobile3.data.model;

import com.squareup.moshi.Json;

public class Etudiant {

    @Json(name = "d")
    private EtudiantData etudiantData;

    public EtudiantData getEtudiantData() {
        return etudiantData;
    }

    public void setEtudiantData(EtudiantData etudiantData) {
        this.etudiantData = etudiantData;
    }
}

File: app/src/main/java/ca/etsmtl/etsmobile3/data/model/Etudiant.kt
package ca.etsmtl.etsmobile3.data.model

import com.squareup.moshi.Json

class Etudiant: SignetsModel() {
    @Json(name = "d")
    var data: EtudiantData? = null

    override fun getSignetsData(): SignetsData? {
        return data
    }
}
